libraries/reconstruction/resolution_monotomo.cpp

```diff
@@ -481,138 +518,139 @@ void ProgMonoTomo::localNoise(MultidimArray<double> &noiseMap, Matrix2D<double>
 				yLimit = STARTINGY(noiseMap) + (Y_boxIdx+1)*boxsize;
 			}
 
+
+
 			counter = 0;
 			for (int i = (yStart); i<(yLimit); i++)
```

Comment:
This is very ineffective code. You are basically taking every 257th element of your volume, which has cubic complexity. You can rewrite the code to take them directly, which would be linear complexity.